sympy/solvers/diophantine.py

```diff
@@ -2170,17 +2175,35 @@ def parametrize_ternary_quadratic(eq):
     Examples
     ========
 
+    >>> from sympy import Tuple, ordered
     >>> from sympy.abc import x, y, z
     >>> from sympy.solvers.diophantine import parametrize_ternary_quadratic
-    >>> parametrize_ternary_quadratic(x**2 + y**2 - z**2)
-    (2*p*q, p**2 - q**2, p**2 + q**2)
+    >>> parametrize_ternary_quadratic(4*x**2 + 2*y**2 - 3*z**2)
+    (2*p**2 - 3*q**2, -4*p**2 + 12*p*q - 6*q**2, 4*p**2 - 8*p*q + 6*q**2)
+
+        More than one solution can be represented by a given ``p`` and
```

Comment:
Is the extra indentation necessary?

Reply:
My feeling was that this is more of a parenthetical comment and I wanted it to appear as such. Perhaps putting it under Notes would be better?

Reply:
That might be better unless the presence of examples will confuse Sphinx.